include.xml

```diff
@@ -6,8 +6,9 @@
 	<assets path="assets/graphics" rename="graphics" include="*.png" embed="true"/>
 	<assets path="assets/font" rename="font" include="*.ttf" embed="true"/>
 
-	<set name="legacy" unless="next"/>
-	<set name="openfl-legacy" unless="next || hybrid"/>
+	<haxedef name="tile_shader" if="openfl > 4.0"/>
```

Comment:
That'll work. I'd like to have a specific flag for this but will go with this for now to have something working.